en/micro-integrator/docs/use-cases/examples/rabbitmq_examples/requeue-msgs-with-errors-rabbitmq.md

```diff
@@ -1,7 +1,8 @@
 # Requeue a message preserving the message order with a delay in case of error
 
-!!! Note
-    <b>Work in progress!</b>
+This sample demonstrates how you can requeue a message to the RabbitMQ queue which was consumed by a consumer proxy,
+in a case on an error. In the sample given below, if the HTTP endpoint becomes unavailable the message will be put back
```

Comment:
in case of an error? when an error occurs?

Reply:
+1. changed to when an error occurs